app/src/main/java/swati4star/createpdf/util/SplitPDFUtils.java

```diff
@@ -30,13 +28,18 @@
     private static final int ERROR_PAGE_RANGE = 2;
     private static final int ERROR_INVALID_INPUT = 3;
 
-    private final Activity mContext;
-    private SharedPreferences mSharedPreferences;
+    private static Activity mContext;
```

Comment:
Isnt this going to cause memory leaks?

Reply:
Why it would? Because the `final` was removed?

Reply:
because you made it static

Reply:
Remove mContext from constructor & get it as parameter to relevant functions